### PR TITLE
update release plan

### PR DESCRIPTION
**Pull request by DavidKutu**
-   [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [ ] Title summarizes what is changing.
-   [ ] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [ ] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -1132,16 +1132,3 @@ jobs:
         run: |
           echo ${{steps.version.outputs.version}}
         shell: bash
-
-      - name: Create GH Release
```
Comment (DavidKutu) [inline .github/workflows/build-test.yml:1136]: for security reasons, we removed the token and this fails. We can do this step manually

File: .github/release_plan.md [hunk 2]
```diff
@@ -56,7 +56,8 @@
     -   [ ] Create pull request against `release-YYYY.MM` (🤖)
     -   [ ] Merge pull request into `release-YYYY.MM`
 -   [ ] Make sure component governance is happy
--   [ ] Turn off automatic uploads for insider builds from main. This prevents stable customers from getting insiders builds as they have the same engine version and higher build numbers.
+-   [ ] [Turn off automatic uploads for insider builds from main](https://github.com/microsoft/vscode-jupyter/blob/f05fedf399d34684b408245ba27bc29aa25c13f6/.github/workflows/build-test.yml#L73). This prevents stable customers from getting insiders builds as they have the same engine version and higher build numbers.
+-   [ ] Do a quick sanity check on the appropiate VS Code version (try to get the candidate build from VS Code, if not available, build the release branch locally, if there's no release branch, use the latest indiers).
```
Comment (IanMatthewHuff) [inline .github/release_plan.md:60]: appropriate and insiders.

**Review by IanMatthewHuff — APPROVED** [review]
Thanks for updating.

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (IanMatthewHuff) [reply .github/workflows/build-test.yml:1136]: @joyceerhl  didn't you restore permissions for a few actions recently? Can we do that here?

Comment (joyceerhl) [reply .github/workflows/build-test.yml:1136]: Yep, you would add the following near the top of this file:
```
permissions:
  pull-requests: write
```

**Pull request merged**